In [8]:
import pandas as pd

df=pd.read_csv("/Users/cyrilmahfouz/Desktop/SEF/Week2/kc_house_data.csv")
df.head(5)
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 21613 entries, 0 to 21612
Data columns (total 21 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   id             21613 non-null  int64  
 1   date           21613 non-null  object 
 2   price          21613 non-null  float64
 3   bedrooms       21613 non-null  int64  
 4   bathrooms      21613 non-null  float64
 5   sqft_living    21613 non-null  int64  
 6   sqft_lot       21613 non-null  int64  
 7   floors         21613 non-null  float64
 8   waterfront     21613 non-null  int64  
 9   view           21613 non-null  int64  
 10  condition      21613 non-null  int64  
 11  grade          21613 non-null  int64  
 12  sqft_above     21613 non-null  int64  
 13  sqft_basement  21613 non-null  int64  
 14  yr_built       21613 non-null  int64  
 15  yr_renovated   21613 non-null  int64  
 16  zipcode        21613 non-null  int64  
 17  lat            21613 non-null  float64
 18  long  

In [7]:
lat=df.lat.mean()
long=df.long.mean()

print("avrg lat of houses: ",lat)
print("avrg long of houses: ",long)

print(len(df))

avrg_area=df.sqft_lot.mean()+df.sqft_living.mean()
print("the avrg area of the houses: ",avrg_area)

print("the avrg building year of the houses: ",df.yr_built.mean())



avrg lat of houses:  47.56005251931708
avrg long of houses:  -122.21389640494147
21613
the avrg area of the houses:  17186.867302086706
the avrg building year of the houses:  1971.0051357978994


To start,we are looking at a dataframe representing house sales in King County, USA from Kaggle. To start this dataframe contains information about 21613 houses.We can see that these house revolves around a latitude of 47.56 and a longitude of -122.21.Also by calculating the averge of the sqft living and the sqft lot we can see that the average area of these houses is 17186.86 meter squarred and the average houses are built in the year 1971